src/cmd/viper.go

```diff
@@ -69,14 +72,12 @@ const (
 	V_PKG_DEPLOY_PUBLIC_KEY = "package.deploy.public_key"
 
 	// Package publish config keys
-	V_PKG_PUBLISH_OCI_CONCURRENCY      = "package.publish.oci_concurrency"
 	V_PKG_PUBLISH_SIGNING_KEY          = "package.publish.signing_key"
 	V_PKG_PUBLISH_SIGNING_KEY_PASSWORD = "package.publish.signing_key_password"
 
 	// Package pull config keys
-	V_PKG_PULL_OCI_CONCURRENCY = "package.pull.oci_concurrency"
-	V_PKG_PULL_OUTPUT_DIR      = "package.pull.output_directory"
-	V_PKG_PULL_PUBLIC_KEY      = "package.pull.public_key"
+	V_PKG_PULL_OUTPUT_DIR = "package.pull.output_directory"
+	V_PKG_PULL_PUBLIC_KEY = "package.pull.public_key"
```

Comment:
Something like this will maintain the output directory for backwards compat:
```
outputDirectory := v.GetString("package.create.output_directory")
	output := v.GetString("package.create.output")
	if outputDirectory != "" && output == "" {
		v.Set("package.create.output", outputDirectory)
	}
```